### PR TITLE
Deleted a test and a minor api addition to MobiusTestFixture

### PR DESCRIPTION
**Pull request by ragunathjawahar**

### CODE REVIEW

File: app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityModelProducerTest.kt [hunk 1]
```diff
@@ -98,37 +97,6 @@ class AllPatientsInFacilityModelProducerTest {
     }
   }
 
-  @Ignore("This is automatically handled by Mobius, delete this test after manual verification.")
-  @Test
-  fun `when the screen is restored, then the ui change producer should emit the last known state`() {
```
Comment (vinaysshenoy) [inline app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityModelProducerTest.kt]: 👏

File: app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityLogicTest.kt [hunk 2]
```diff
@@ -32,12 +33,28 @@ class AllPatientsInFacilityLogicTest {
...
+  private lateinit var fixture: MobiusTestFixture<AllPatientsInFacilityModel, AllPatientsInFacilityEvent, AllPatientsInFacilityEffect>
+
   @Before
   fun setUp() {
     whenever(userSession.requireLoggedInUser())
         .thenReturn(Observable.just(user))
     whenever(facilityRepository.currentFacility(user))
         .thenReturn(Observable.just(facility))
+
+    fixture = MobiusTestFixture(
```
Comment (vinaysshenoy) [inline app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityLogicTest.kt]: Would it be possible to extract this into a separate `TestRule` that will automatically dispose the fixture? We could also try this out after we migrate another feature so we know what it needs to look like.

Comment (ragunathjawahar) [reply app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityLogicTest.kt]: Let's do this after we migrate another feature.

Comment (vinaysshenoy) [reply app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityLogicTest.kt]: Discussed offline: adding to the list of things to do.

**Pull request merged**